SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestLoadBalancingData/SpatialTestLoadBalancingData.cpp

```diff
@@ -0,0 +1,233 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialGDK/SpatialTestLoadBalancingData/SpatialTestLoadBalancingData.h"
+
+#include "EngineClasses/SpatialNetDriver.h"
+#include "EngineClasses/SpatialPackageMapClient.h"
+
+#include "SpatialCommonTypes.h"
+#include "SpatialConstants.h"
+#include "Test1x2GridStrategy.h"
+
+#include "EngineClasses/SpatialWorldSettings.h"
+#include "Kismet/GameplayStatics.h"
+
+void USpatialTestLoadBalancingDataTestMap::CreateCustomContentForMap()
+{
+	Super::CreateCustomContentForMap();
+
+	Cast<ASpatialWorldSettings>(World->GetWorldSettings())
```

Comment:
What if the Cast fails?

Reply:
These are the world settings required for the Spatial to work; a user could break this if they really want, but I'm not sure if we should guard against this.